examples/OSWEC/wecSimInputFile.m

```diff
@@ -11,21 +11,22 @@
 simu.CITime = 30;                       % Specify CI Time [s]
 
 %% Wave Information
+
 % % noWaveCIC, no waves with radiation CIC  
 % waves = waveClass('noWaveCIC');       % Initialize Wave Class and Specify Type  
 
 % % Regular Waves 
-% waves = waveClass('regular');           % Initialize Wave Class and Specify Type                                 
-% waves.H = 2.5;                          % Wave Height [m]
-% waves.T = 8;                            % Wave Period [s]
+waves = waveClass('regular');           % Initialize Wave Class and Specify Type                                 
+waves.H = 2.5;                          % Wave Height [m]
+waves.T = 8;                            % Wave Period [s]
 
 % Irregular Waves using PM Spectrum with Directionality 
-waves = waveClass('irregular');         % Initialize Wave Class and Specify Type
-waves.H = 2.5;                          % Significant Wave Height [m]
-waves.T = 8;                            % Peak Period [s]
-waves.spectrumType = 'PM';              % Specify Spectrum Type
-waves.waveDir = [0,30,90];              % Wave Directionality [deg]
```

Comment:
@salhus If possible can you run the OSWEC example with lines 27 and 28 uncommented, or rather can you run with the directional spreading active? This would help to see how the balls would float with a more complex environment and answer our question if directional spreading is being used in the calculation of the wave gauge wave elevation.

Reply:
![irr_wave_with_spreading](https://user-images.githubusercontent.com/84348506/129951113-011f8f1a-aa1b-4eba-a4e7-6c5959892499.gif)
@nathanmtom Thanks for pointing me there.. It seemed to work actually..

Reply:
@nathanmtom Forgot to write in the last comment. I commented out the regular wave code and uncommented lines 23 - 29. Line 29 sets up the spreading, so seems like it did work..